build.py

```diff
@@ -1243,30 +1243,33 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
 
 # Common dependencies. FIXME (can any of these be conditional? For
 # example libcurl only needed for GCS?)
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-            software-properties-common \
-            libb64-0d \
-            libcurl4-openssl-dev \
-            libre2-9 \
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+            clang \
```

Comment:
@rmccorm4 I wasn't able to run it against `ubuntu:22.04` image without `clang` package preinstalled.